.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -1,57 +1,75 @@
----
 name: Bug Report
-about: Report a bug in the .NET MAUI Toolkit
-title: "[Bug] "
-labels: bug, unverified
-assignees: ''
-
----
-
-<!--
-Hello, and thanks for your interest in contributing to the .NET MAUI Toolkit! 
-
-Please describe the issue below, including detailed steps for reproduction. If we are unable to reproduce the bug we will be unable to fix the issue and may close this Issue.
-
--->
-
-### Description
-
-<!-- Please provide a detailed description of the issue -->
-
-### Stack Trace
-
-<!-- Please copy/paste the complete stack trace (if applicable) -->
-
-### Link to Reproduction Sample
-
-<!-- Please link to a GitHub Repo or GitHub Page -->
-
-### Steps to Reproduce
-
-1. 
-2. 
-3. 
-
-### Expected Behavior
-
-### Actual Behavior
-
-### Basic Information
-
-- Version with issue:
-- Last known good version:
-- IDE:
-- Platform Target Frameworks: <!-- All that apply -->
-  - iOS:  <!-- The version of the iOS SDK you are compiling against, e.g. 11.1 -->
-  - Android: <!-- The version of the Android SDK you are compiling against, e.g. 7.1 --> 
-  - UWP:  <!-- The version of the UWP SDK you are compiling against, e.g. 16299 --> 
-- Android Support Library Version: <!-- if applicable -->
-- Nuget Packages:
-- Affected Devices:
-
-### Workaround
-<!-- If you've found a way around this issue, help other users out and share the workaround -->
-
-### Reproduction imagery
-
-<!-- If it is a visual issue, please include imagery (GIF/Screenshots) showing the problem -->
+description: Report a bug in the .NET MAUI Toolkit
+title: "[BUG] <title>"
+labels: [bug, unverified]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. Open and run solution fron reproduction repository.
+      1. Click the button and observe the bug 🐞
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Link to public reproduction project repository
+    description: Please add a link to a public [reproduction project](https://github.com/dotnet/maui/blob/main/.github/repro.md) repository. Otherwise the issue will be marked with "Needs reproduction" label and will have a lower priority. Attached zip files cannot be opened by us.
+  validations:
+    required: true
+- type: dropdown
```

Comment:
I'm wondering if having the Community Toolkit version in a drop down will mean we have to keep remembering to update this bug template with each release.  Also, if we are requiring a repro project, wouldn't we be able to see the version of the toolkit from that?

Reply:
even if we ask a user to provide the link, there is no guarantee he will do it (Or he can use the path to the source instead of using nuget package). Having a version may help us to figure out what version is affected.

I can rename it to affected versions and allow multiple selections.

Reply:
Sorry what I meant was if the Community Toolkit version number is a drop down with values - won't we have to update the bug template every release to add the new version number to the drop down values.

Should it just be an entry field?

Reply:
yes, we'll need to update the template. I found it is used in .NET MAUI repo. Probably it would be better to have a similar form. But I have no objections if we choose input field

Reply:
I'm OK to try with a field. If we see the community members are having difficulties with that we can change to be a drop down

Reply:
> I found it is used in .NET MAUI repo

To be honest I'm thinking of removing it there, exactly because we constantly need to remember to upgrade it. Additional problem with .NET MAUI is t hat people have no clue which version they are actually using most of the time. I'm not sure what a good alternative is though, but that's a whole different discussion :D

Reply:
I removed that field and combined it with Environment. At least they should know the OS version